app/src/main/java/org/dhis2/usescases/teiDashboard/DashboardRepositoryImpl.kt

```diff
@@ -587,6 +587,29 @@ class DashboardRepositoryImpl(
         )
     }
 
+
+    override fun transferTei(newOrgUnitId: String): Single<Boolean> {
+        return d2.trackedEntityModule()
+            .ownershipManager()
+            .transfer(teiUid, programUid!!, newOrgUnitId)
+            .andThen(Single.just(true))
```

Comment:
Why do you need a boolean?. This `transfer` shouldn't fail. May just perform the action withou return value?

Reply:
Done